editor/editor_node.cpp

```diff
@@ -1130,6 +1131,10 @@ void EditorNode::save_resource_in_path(const Ref<Resource> &p_resource, const St
 	}
 
 	String path = ProjectSettings::get_singleton()->localize_path(p_path);
+	DirAccess *da = DirAccess::open("res://");
```

Comment:
You should probably protect the remaining of the function from crash with `ERR_FAIL_COND(!da)`.